### DISCUSSION TITLE
Multi box band

### DISCUSSION THREAD

**Discussion by HighlanderMisfits** [akira0245/MidiBard#25] [Ideas]
Super interested in your plugin, it always make life so much simpler with ensemble mode integration.

However, I have an issue when multi-boxing my 8 piece band. There is currently no way to automatically switch the whole band to a new track, or switch to the correct instruments after selecting the track like BMP does. This makes use of the vanilla release rather difficult.

Is it possible to have a command to switch all bards in the same group to the same track and automatically change the instruments to the correct ones?

Ultimately, I'm trying to migrate away from BMP and use your plugin but I'm looking for those smaller tweaks to make it compatible with my BMP midi files. I have hundreds of midi files so I don't really want to edit them all with new track names to enable auto instrument switching if it can be helped.

**Comment by akira0245** [comment]
Hmmm... maybe i should add something like local orchestra. I didn't do it before because i initially want to communite across different computers, and that requires a server or suspicious party chat, which added a lot more uncertainty. 
The auto instrument switching based on track name is already implemented, just not in release version yet. 
I'll try release a beta version which has IPC based party communication in a few days.
